src/ngx_lua_resty_lmdb_module.c

```diff
@@ -363,6 +375,151 @@ ngx_lua_resty_lmdb_verify_file_status(ngx_cycle_t *cycle,
 }
 
 
+static ngx_int_t
+ngx_lua_resty_lmdb_validate(ngx_cycle_t *cycle,
+                            ngx_lua_resty_lmdb_conf_t *lcf)
+{
+    int                        rc;
+    MDB_dbi                    dbi;
+    MDB_val                    key;
+    MDB_val                    value;
+    MDB_txn                   *txn = NULL;
+
+    ngx_str_t                  validation_key =
```

Comment:
We can get rid of this temporary variable, it doesn't seems to be useful.

Reply:
done